src/main/java/dev/jbang/source/ScriptSource.java

```diff
@@ -521,6 +707,12 @@ public static ScriptSource prepareScript(String resource) {
 	}
 
 	public static ScriptSource prepareScript(ResourceRef resourceRef) {
-		return new ScriptSource(resourceRef);
+		String originalResource = resourceRef.getOriginalResource();
+		System.out.println("originalResource = " + originalResource);
```

Comment:
this system out breaks jbang with error as its executed.

Reply:
stupid debug.  one moment.

Reply:
this is gone now